x-pack/elastic-agent/pkg/agent/application/info/agent_id.go

```diff
@@ -26,8 +26,11 @@ const agentInfoKey = "agent"
 // defaultAgentActionStoreFile is the file that will contains the action that can be replayed after restart.
 const defaultAgentActionStoreFile = "action_store.yml"
 
+const defaultLogLevel = "debug"
```

Comment:
With this change can we move the default up to info?

Debug level is still causing duplicate events, which still needs to be solved even in debug.